pkg/operator/startupmonitorreadiness/readiness_checks.go

```diff
@@ -377,3 +379,17 @@ func filterByNodeName(kasPods []corev1.Pod, currentNodeName string) []corev1.Pod
 
 	return filteredKasPods
 }
+
+func getExcludedChecks() []string {
```

Comment:
should we also exclude the poststarthook for the openshift readiness?

https://github.com/openshift/kubernetes/blob/53f3066f47d3a594d6b93843916e8711ee673715/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```
	genericConfig.AddPostStartHookOrDie("openshift.io-openshift-apiserver-reachable", func(context genericapiserver.PostStartHookContext) error {
		go openshiftAPIServiceReachabilityCheck.checkForConnection(context)
		return nil
	})
```

Reply:
Yeah, we may want to exclude this and `oauth-server` too - let me add those too